src/components/dialog/angularjs/dialog_directive.js

```diff
@@ -193,6 +160,40 @@ function DialogController(
         }, _TRANSITION_DURATION);
     }
 
+    /**
+     * Create dialog content observer.
```

Comment:
Maybe you should be a little bit more exhaustive about what you do with those observers